pages/user guides/migration_waves.md

```diff
@@ -0,0 +1,72 @@
+
+## Prerequisites
+
+Migration waves exist in projects, and function as a way to break a large project down into logical chunks. In order to proceed, you should first create a project in your workspace. For guidance on how to create a project, see [creating a project](assessment-projects.html#creating-a-project).
+
+## Creating migration waves
+
+To create a migration wave, first navigate to the project where you'd like to create this wave. 
+
+On the project page, you should see a tab called "Migration Waves".
+
+{% include image.html file="migration-waves-1.png" %}
+</br>
+
+In this tab, you will see all existing waves of this project. To create a new wave, click "Create Wave". 
+
+{% include image.html file="migration-waves-2.png" %}
+</br>
+
+Enter a name and - optionally - a time frame for the migration wave. 
+
+{% include image.html file="migration-waves-3.png" %}
+</br>
+
+After the migration wave is created, it should appear in the table. You are now ready to assign some resources to this wave.
+
+## Assigning Resources to a Migration Wave
+
+From the "Migration Waves" project tab, click the name of the wave you'd like to assign resources to. You will see an overview of the migration wave, as well as tabs for applications, servers and database instances.
+
+The process for assigning resources to a wave is the same across all resource types. For this guide, we will assign some applications to the migration wave.
+
+Click on the "Apps" tab of your migration wave. You should see a table with the button "Add Applications".
+
+{% include image.html file="migration-waves-4.png" %}
+</br>
+
+After clicking this button, you will be taking to a staging view. Here you can select which applications you'd like to assign to the wave. You can search your applications by name or tag, then make your selection by using the checkboxes in the application table. Once you've made your selection, click "Stage".
+
+{% include image.html file="migration-waves-5.png" %}
+</br>
+
+The purpose of a migration wave is to allow you to define groups of resources which make logical sense to migrate together. for example - if you choose to migrate a given application, it makes sense to also include that application's servers and database instance in the same migration wave.
+
+On this page, you can see the dependant applications, servers and database instances of your staged resources. If required, you can use the table along the bottom of the page to add these dependant resources to the migration wave. The pie chart on the top left will be updated as you go, allowing you to see the breakdown of your wave by resource type and environment.
+
+{% include image.html file="migration-waves-6.png" %}
+</br>
+
+By default, it is not possible to stage dependant resources which are already in other migration waves. If you'd like to do this, there is a toggle on the top right of the page labelled 'Allow assignment to multiple migration waves'. 
```

Comment:
IIRC, it's not just dependent resources; this includes the staged resources as well
> In most cases, having a record spread across multiple migration waves is unnecessary and confusing. But there are cases when a record needs to be staged across multiple waves, so we give you the escape hatch through this button.